src/com/sap/piper/JsonUtils.groovy

```diff
@@ -1,8 +1,20 @@
 package com.sap.piper
 
 import com.cloudbees.groovy.cps.NonCPS
+import groovy.json.JsonBuilder
+import groovy.json.JsonSlurperClassic
+
+@NonCPS
+def jsonToString(content) {
+    return new JsonBuilder(content).toPrettyString()
+}
 
 @NonCPS
 String getPrettyJsonString(object) {
     return groovy.json.JsonOutput.prettyPrint(groovy.json.JsonOutput.toJson(object))
 }
+
+@NonCPS
+def parseJsonSerializable(text) {
```

Comment:
This name is misleading: The input is not a json serializable (i.e. something which can be converted into json) but a json document, which you are going to deserialize.